dotnet/src/SemanticKernel/Kernel.cs

```diff
@@ -301,28 +301,30 @@ public void Dispose()
     private static Dictionary<string, ISKFunction> ImportSkill(object skillInstance, string skillName, ILogger logger)
     {
         MethodInfo[] methods = skillInstance.GetType().GetMethods(BindingFlags.Static | BindingFlags.Instance | BindingFlags.Public);
-        logger.LogTrace("Importing skill name: {0}. Potential methods found: {1}", skillName, methods.Length);
-
+        logger.LogTrace($"Importing skill name: {skillName}. Potential methods found: {methods.Length}");
```

Comment:
Revert this change.

See the following for the reason why:
https://blog.objektkultur.de/structured-logging-with-.net-ilogger/